src/MPQEditor/MPQEditor.ts

```diff
@@ -74,6 +75,36 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
     return undefined;
   }
 
+  /**
+   * @brief A helper function to find unoccupied mpq file-name
+   * @returns valid file name for mpq configuration or an empty string on failure
```

Comment:
Q) above `validateMPQName` returns `undefined` on failure. this returns empty string. is there any particular reason not to return `undefined` here too?

Reply:
> Q) above `validateMPQName` returns `undefined` on failure. this returns empty string. is there any particular reason not to return `undefined` here too?

Ahh. Sorry. I'll fix it.

Reply:
@seanshpark 
fixed